app/views/admins/service_requests/edit.html.haml

```diff
@@ -63,18 +63,20 @@
         .row.well
           %h4= t('.update_service_request')
           = form_for [:admins, @service_request] do |f|
-            = f.object.errors.messages unless f.object.errors.empty?
-            .form-group
-              = f.label :public_servant_id, t('form.labels.admin.who')
-              - if @service_request.public_servant_id.nil? || @service_request.public_servant_id == 0
-                = f.select :public_servant_id, @admins_services.collect {|p| [ p.name,  p.id] }, { :include_blank =>  t('service_requests.form.prompt_who') }, { class: 'js-load_service_fields form-control' }
-                = t('form.labels.admin.description')
-                = @service_request.public_servant_description
-              -else
-                -unless current_admin.is_public_servant?
-                  = f.select :public_servant_id, @admins_services.collect {|p| [ p.name,  p.id] }, { prompt: t('service_requests.form.prompt_who'), :selected => Admin.find(@service_request.public_servant_id).id }, { class: 'js-load_service_fields form-control' }
+            -unless f.object.errors.empty?
+              = f.object.errors.messages 
+            -unless current_admin.is_service_admin? || current_admin.is_public_servant?
```

Comment:
Line is too long. [87/80]<br>The - symbol should have one space separating it from code